src/general/HybridAudioPlayer.cs

```diff
@@ -0,0 +1,110 @@
+﻿using Godot;
+
+/// <summary>
+///   Provides extra level of abstraction to allow simultaneous switching between 3D positional and non positional
+///   audio players in a single Node-derived class.
+/// </summary>
+/// <remarks>
+///   <para>
+///     Useful in cases where the playing of an audio stream must be able to fulfill both of these conditions.
+///   </para>
+/// </remarks>
+public class HybridAudioPlayer : Node
+{
+    private AudioStreamPlayer3D player3D;
+    private AudioStreamPlayer playerNonPositional;
+
+    private float volume;
+
+    public HybridAudioPlayer()
+    {
+        player3D = new AudioStreamPlayer3D();
+        playerNonPositional = new AudioStreamPlayer();
+
```

Comment:
Would it be completely crazy to suggest that these would be created on demand? With this approach we create potentially hundreds of unnecessary audio player objects.

Reply:
No that's perfectly sensible, I think Positional property should be readonly.

Reply:
Added.